packages/@sanity/cli/test/shared/environment.ts

```diff
@@ -56,9 +57,9 @@ const getTestId = () => {
     return cachedTestId
   }
 
-  const localId = readFileSync(testIdPath, 'utf8').trim()
+  const localId = readFileSync(testIdPath, 'utf8').trim().slice(0, 5)
```

Comment:
yes, it for machines like mine with super long name since the dataset name is capped at 64 characters. It just makes it shorter so it can run locally